src/theory/quantifiers/ematching/inst_match_generator_multi.cpp

```diff
@@ -163,23 +163,15 @@ uint64_t InstMatchGeneratorMulti::addInstantiations(Node q)
     std::vector<InstMatch> newMatches;
     InstMatch m(q);
     while (d_children[i]->getNextMatch(q, m) > 0)
```

Comment:
it's a bit counterintuitive both that this returns an int and that it's tested like this. It'd be good to change to `bool` eventually

Reply:
Agreed